frontend/src/schema.graphql

```diff
@@ -120,6 +122,7 @@ input NewVideoBlock {
 interface Block {
   id: ID!
   index: Int!
+  realm: Realm!
```

Comment:
We talked about this field in the past. There is potential for "abuse" here, because sometimes using this is way easier (and simpler!) than structuring your code in a way so that all the block code knows about the realm it is in from a "higher up" query. But of course that's slow because we don't optimize queries across type boundaries, yet.

An alternative for the use cases you use this in is to introduce a new type that contains a block and its realm side by side, which you can then return from mutations. See the `removeBlock` mutation.

Reply:
While I still think the same way about the code in question when we talked about it back then, I don't really think we need to protect ourselves against accidentally using a slow field. We have lots of these already. And in the specific case I use it, I think it's nicer and maybe even more idiomatic to do it like I did. Return a (block, realm) tuple from the mutation where `Block.realm` is such an obvious field (due to their relationship) feels weird.

Reply:
Maybe. I mean honestly, I **want** this field, I just also want it to be fast. x)